lib/haml_lint/linter/indentation.rb

```diff
@@ -9,15 +9,47 @@ class Linter::Indentation < Linter
       tab: /^\t*(?![ ])/,
     }.freeze
 
+    LEADING_SPACES_REGEX = /^( +)(?! )/
+
     def visit_root(root)
-      regex = INDENT_REGEX[config['character'].to_sym]
+      character = config['character'].to_sym
+      check_character(character, root)
+
+      width = config['width'].to_i
+      check_width(width, root) if character == :space && width > 0
```

Comment:
Oh, I see. You're only checking the width for spaces, not tabs.

I think this is probably okay since I don't see "2 tab indents" being a thing, but it might be confusing if someone tries it out.